README.md

```diff
@@ -1 +1,24 @@
-# Recipe-app
\ No newline at end of file
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
```

Comment:
- [ ] As a requirement for this project, you are required to have a professional `README.md`. Please add al the main headers as seen in the default [template](https://github.com/microverseinc/readme-template). Customize to suit all the features of your project.

Reply:
Thanks for your feedback. 
Readme file Added ✅